styles/header.css

```diff
@@ -0,0 +1,79 @@
+.header {
+  display: flex;
+  height: 48px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.left-section {
+  margin-left: 12px;
+  margin-top: 4px;
+  margin-bottom: 4px;
+}
+
+.logo {
+  color: #6070ff;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  padding: 10px 12px;
+  text-decoration: none;
+}
+
+.right-section {
+  margin-right: 24px;
+  margin-top: 4px;
+  margin-bottom: 4px;
+  border: none;
+  background-color: white;
+}
+
+.icon-menu {
+  padding: 8px;
+  width: 24px;
+  height: 24px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  margin-left: 24px;
+  margin-right: 24px;
+  background-image: url('/header-shapes\ mobile@2x.svg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
```

Comment:
- [ ] Woo :+1: You have implemented most changes requested by the previous reviewer and you now have a nice layout, What is left is for you to add a background image to your section  :blush: to match the [ Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) :blush:
![Screenshot from 2022-12-01 16-18-49](https://user-images.githubusercontent.com/92303891/205105729-5ba25800-2110-45b9-8976-533b7cd6ed33.png)

Reply:
Thank you